cmd/marvelassignment/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/joho/godotenv"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	err := godotenv.Load("/Users/taylor.hottel/pivotassignments/cmd/marvelassignment/.env")
```

Comment:
Again, this only works for you and only on your machine. Do not hardcode paths like this.